source/administration/command-line-tools.rst

```diff
@@ -358,6 +359,29 @@ mattermost channel restore
 
       ./mattermost channel restore 8soyabwthjnf9qibfztje5a36h
       ./mattermost channel restore myteam:mychannel
+      
+mattermost channel search
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Search for a channel by channel name. 
```

Comment:
What's the output? Is it to search if a channel already exists? Or to return the channel ID?

Reply:
updated the description to include the outputs.